### DISCUSSION TITLE
Laravel Laraberg Support

### DISCUSSION THREAD

**Discussion by safarovitch** [WordPress/gutenberg#34427] [Q&A] [answered]
Hi,
Does this package work with laraberg?

**Comment by youknowriad** [comment, marked as answer]
Hi there!

The Gutenberg repository provides generic package (notably @wordpress/block-editor) that other developers can adapt to multiple platforms including Laravel. 

I believe the [Laraberg](https://github.com/VanOns/laraberg) is the integration you're talking about. If you're having trouble using it, I'd suggestion asking on the dedicated repository instead. https://github.com/VanOns/laraberg 

Thanks.
